packages/itinerary-body/src/AccessLegBody/mapillary-button.tsx

```diff
@@ -96,14 +96,17 @@ const MapillaryButton = ({
   };
 
   if (!imageId) return null;
+  const mapilaryLabel = intl.formatMessage({
+    defaultMessage: defaultMessages["otpUi.AccessLegBody.mapillaryTooltip"],
+    description: "Tooltip text describing the street view icon.",
+    id: "otpUi.AccessLegBody.mapillaryTooltip"
+  });
   return (
     <Container
       onClick={handleClick}
-      title={intl.formatMessage({
-        defaultMessage: defaultMessages["otpUi.AccessLegBody.mapillaryTooltip"],
-        description: "Tooltip text describing the street view icon.",
-        id: "otpUi.AccessLegBody.mapillaryTooltip"
-      })}
+      title={mapilaryLabel}
+      aria-label={mapilaryLabel}
```

Comment:
Do you have confirmation we need both of these?

Reply:
Screenreaders can't consistently see titles, but sighted users can't hover to see aria-labels, so I think there's a benefit in both, especially for a link where the icon isn't inherently clear!